extensions/amp-analytics/0.1/vendors/taboola.json

```diff
@@ -3,7 +3,7 @@
     "aid": ""
   },
   "requests": {
-    "loadamptrkhtml": "https://c3.taboola.com/amp/amp-analytics-taboola.html?url=${canonicalUrl}&aid=${aid}"
+    "loadamptrkhtml": "https://c3.taboola.com/amp/amp-analytics-taboola.html?url=${canonicalUrl}&aid=${aid}&consent=${consent}"
```

Comment:
Where is this `${consent}` variable defined? Or are you trying to use AMP predefined consent variable substitutions?

Reply:
@micajuine-ho yes using the predefined consent variable substitutions. The idea is our publisher partners will setup the AMP predefined consent variable.

Reply:
I believe you are looking for theses variables: https://github.com/ampproject/amphtml/blob/main/extensions/amp-analytics/0.1/variables.js#L321-L327

Did you see `${consent}` anywhere in our documentation?

Reply:
@micajuine-ho fixed the substitution names. sorry about that.